<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegistrarTrabajador</class>
 <widget class="QDialog" name="RegistrarTrabajador">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>423</width>
    <height>474</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="txtNombre">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>60</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>30</number>
   </property>
  </widget>
  <widget class="QRadioButton" name="rbtHom">
   <property name="geometry">
    <rect>
     <x>116</x>
     <y>180</y>
     <width>69</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Masculino</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>140</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>DNI:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>63</x>
     <y>100</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Apellido:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>63</x>
     <y>217</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Usuario:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>63</x>
     <y>180</y>
     <width>39</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Genero:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>63</x>
     <y>60</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Nombre:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtApellido">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>100</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>30</number>
   </property>
  </widget>
  <widget class="QRadioButton" name="rbtMu">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>180</y>
     <width>69</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Femenino</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>63</x>
     <y>257</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Contra:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtDni">
   <property name="geometry">
    <rect>
     <x>116</x>
     <y>140</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>8</number>
   </property>
  </widget>
  <widget class="QPushButton" name="btnRegistrar">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>380</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Registrar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancelar">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>380</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancelar</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtUsu">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>220</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtContra">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>260</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Codigo:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtCodigo">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>30</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbNivel">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>310</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>310</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Nivel:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
